pyccel/stdlib/ndarrays/ndarrays.c

```diff
@@ -7,18 +7,57 @@
 # include <string.h>
 # include <stdarg.h>
 # include <stdlib.h>
+# include <stdio.h>
 
 /*
 ** allocation
 */
 
+void print_ndarray_memory(t_ndarray nd)

```

Comment:
do you need this function?

Reply:
It has kind of has the same functionality as `np.nditer`, I am not using it anywhere in the code, but it was pretty usefull for checking different memory layouts, and comparing them to `np.nditer`, I am guessing it will probably be used in the future.
Should I just remove it?

Reply:
If it's useful for debugging, then I vote we keep it. But it needs a docstring to explain what it's useful for

Reply:
+1 on what @EmilyBourne said a docstring would be nice